.github/workflows/build-docs.yml

```diff
@@ -0,0 +1,25 @@
+name: Docs
+
+on:
+  push:
+    branches:
+      - master
+  schedule:
+    # <minute [0,59]> <hour [0,23]> <day of the month [1,31]> <month of the year [1,12]> <day of the week [0,6]>
+    # https://pubs.opengroup.org/onlinepubs/9699919799/utilities/crontab.html#tag_20_25_07
+    # Run every Monday at 18:00:00 UTC (Monday at 10:00:00 PST)
+    - cron: '0 18 * * 1'
+
+jobs:
+  docs:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - name: Generate Docs
+      uses: JacksonMaxfield/github-pages-deploy-action-python@master
+      env:
+        ACCESS_TOKEN: ${{ secrets.ACCESS_TOKEN }}
+        BASE_BRANCH: master # The branch the action should deploy from.
+        BRANCH: gh-pages # The branch the action should deploy to.
+        FOLDER: docs/_build/html # The folder the action should deploy. This example folder is generated by Sphinx
+        BUILD_SCRIPT: pip install .[dev] && make docs && touch docs/_build/html/.nojekyll # The build script the action should run prior to deploying.
```

Comment:
Why the `touch .nojekyll`?

Reply:
`.nojekyll` tell's github pages to not do anything with these files we are about to deploy. The `gh-pages` branch runs jekyll by default.